src/PerformanceTests/Common/Program.cs

```diff
@@ -83,7 +93,6 @@ static int Main()
             finally
             {
                 Log.Info("Fin!");
-                NLog.LogManager.Shutdown();
```

Comment:
I think log4net also has a shutdown method.

Reply:
There seems to be a LogManager.Shutdown method. The log4net docs is just terrible.

Reply:
I removed the call to shutdown. It's called by log4net when the appdomain exits. Added it back to the cases where we call Environment.FailFast (although I think we should just call Environment.Fail) instead.